### ISSUE TITLE
unable to connect - EUID error 

### ISSUE THREAD

**Issue by loopez76** [jvitkauskas/homeassistant_salus#20]
Same error - "Failed to connect, please check EUID"

IP is pingable from HA (same segment), troubleshooting done, done several reboots. In HA timeout - any ideas ? Same error on 0.3.0 0.3.1

In log of HA:
2021-10-20 17:55:00 ERROR (MainThread) [pyit600] Timeout while connecting to gateway:

SW vewrsion: 020143210405 coordinator 20210317

@jvitkauskas any chance for help ? Multiple resets of gateway did nothing. with GH working fine but GH is connecting over cloud not directly

**Comment by Raoulex** [comment]
Same issue here also.

**Comment by thatvalis** [comment]
Same issue here. Have done a lot of factory resets and/or reconnecting power.
Same error: "Failed to connect, please check EUID"

**Comment by loopez76** [comment]
Still no win. Checked on another gateway with same results. Looks like since some firmware is not working. Integration with GH works fine with no issues

**Comment by ncwp** [comment]
I am getting the same error too, with the timeout error in the log.
Same software coordinator versions as @loopez76.

**Comment by beoneit** [comment]
I also have the same error :/ "Failed to connect, please check EUID" and timeout error in the log.

**Comment by beoneit** [comment]
I just used '0000000000000000' as e EUID and it works.

**Comment by loopez76** [comment]
> 

Just retried with 0000000000000000 as EUID - and confirm. It works.

**Comment by jvitkauskas** [comment]
Thanks, added this suggestion to troubleshooting section of the README

**Comment by ncwp** [comment]
FYI the 0000000000000000 EUID did not work for me.
It looks like my gateway is actually timing out, never responds with any data. If I browse to the gateway in the regular browser I see a lot of requests to `deviceid/read` and they all time out there too. So I guess that maybe this is a problem that cannot be fixed in Python. main.py from pyit600 also times out.

**Comment by pekarsky** [comment]
> FYI the 0000000000000000 EUID did not work for me. It looks like my gateway is actually timing out, never responds with any data. If I browse to the gateway in the regular browser I see a lot of requests to `deviceid/read` and they all time out there too. So I guess that maybe this is a problem that cannot be fixed in Python. main.py from pyit600 also times out.

Oh, I was able to query GW with main.py from py600it, using this EUID
Working on adding integration

**Comment by BrodaBob** [comment]
I don't see any EUID on my gateway. I only see a 15-digit serial #. And that's the same as I can see displayed in the Information menu in the app.
I've tried the 16-zero euid to no avail.
I'm in Europe, btw.
Any advice? :-)

**Comment by RazvanDespa** [comment]
Hi,

I have the same problem, the EUID i see in the app or on the UGE600 gateway has only 15 characters. I have tried also with 16 zeroes but still no luck.

Any idea?

Thanks,
Razvan

**Comment by BrodaBob** [comment]
> Hi,
> 
> I have the same problem, the EUID i see in the app or on the UGE600 gateway has only 15 characters. I have tried also with 16 zeroes but still no luck.
> 
> Any idea?
> 
> Thanks, Razvan

Yeah I figured it out. You're looking at the wrong string. Remove the cables and then you see the correct 16-digit string on the left side :-)

**Comment by RazvanDespa** [comment]
> 

@BrodaBob you are the master ... yes, next there were more codes but due to the position i haven't seen them. All is good now.

**Comment by ConstantinOWORX** [comment]
Make sure to enable the wi-fi local mode, that fixed it for me.

**Comment by alex-ricobon** [comment]
> Make sure to enable the wi-fi local mode, that fixed it for me.

Do I need to connect the gateway to WiFi, or just enable local WiFi mode?

**Comment by trullock** [comment]
Where do I enable Local Wifi mode? It doesn't appear as an option under General...

**Comment by trullock** [comment]
OK, so telneted to the box 
and exec
`/mnt/sal6dg1/run_sal6dg1.sh`
Use the app to change various thermostat settings and you can see a heap of debug information flying by
